README.adoc

```diff
@@ -117,6 +117,27 @@ You should see the following output in your terminal:
 <a href="http://asciidoctor.org"><strong>Asciidoctor</strong></a> running on <a href="http://opalrb.org"><em>Opal</em></a> brings AsciiDoc to Node.js!</p>
 ....
 
+## Node.js examples
+
+This project contain 2 ruby examples `asciidoctor_example.rb` and `userguide_test.rb` which demonstrate how ruby scripts could be converted into javascript using the Opal Runtime
+and next displayed within the browser as a normal HTML file.
+
+The ruby examples are available under the directory `examples` and to build them, a node.js script has been created under the `npm` folder
+
+```
+node npm/examples.js
+```
+
+In order to visualize the result, a local http server must be started within the root of this project otherwise you will have a cross origin issue which prevents to load some resources
+and render the html. You can use for this purpose the node.js module `http-server`.
+
+```
+http-server -a localhost
```

Comment:
I'm using `python -m SimpleHTTPServer` but since this is a JavaScript project, mentioning a Node.js module is better :smile: